client/my-sites/stats/stats-card-upsell/stats-card-upsell-wpcom.tsx

```diff
@@ -30,6 +31,10 @@ const getUpsellCopy = ( statType: string ) => {
 			return translate( 'Identify your audience’s favorite writers and perspectives.' );
 		case STATS_FEATURE_DATE_CONTROL:
 			return translate( 'Compare different time periods to analyze your site’s growth.' );
+		case STAT_TYPE_VIDEO_PLAYS:
+			return translate(
+				'Discover your most popular videos to better understand how they performed.'
+			);
```

Comment:
I've added new copy here, based on the text from the VideoPress upsell and following the style of the other CTA's. Is there someone / team to ping to approve new copy like this?

Reply:
I think it looks good. The only other thought I have is that I'm wondering if we should emphasize the fact that they will at least need the Explorer plan to upload a video first 🤔 

For copy review, we now have pfGA22-2-p2. However, since it's just a one-line copy, I'm wondering if it'd be an overkill. @michaelpick could you help us clarify what should we do if we want a quick copy check like this? I'd be grateful :)

Reply:
Hey @southp! 

> However, since it's just a one-line copy, I'm wondering if it'd be an overkill. @michaelpick could you help us clarify what should we do if we want a quick copy check like this? I'd be grateful :)

Happy to take a look at super-short one-liners in-situ, but single lines of copy have a way of turning into involved discussions, so it's still generally really helpful if folks can submit a review request as it helps us get things on our radar, track work, and stay on top of prioritization!

**A few thoughts for this one!**

I'm not 100% that 'to better understand how they performed' is a logical follow-up to 'discover your most popular videos', but might work with an 'and' instead?

'Discover your most popular videos and understand how they performed.'

A slight rephrasing gives us:

'Discover your most popular videos and find out how they performed.'

A couple of alts:

- Discover your most popular videos and find out what your audience is responding to.
- Find out what's resonating with your audience—discover your most popular videos.


You could then add some subscript/smaller point sized copy below the button, like:

[[Upgrade plan]]
<sub><sup>Available on the Explorer plan and above.</sub><sup>

Reply:
> it's still generally really helpful if folks can submit a review request as it helps us get things on our radar, track work, and stay on top of prioritization!

@michaelpick That makes sense -- will follow this approach for future requests.

Thanks for the copy suggestion, I've updated it to:

> 'Discover your most popular videos and find out how they performed.'

Reply:
@oswian The new string doesn't have a translation and will appear in English for non-EN locales. Can we make it so that the text has a fallback to a translated string?

Reply:
@niranjan-uma-shankar Ah, thank you for pointing this out. `hasTranslation` would be relevant here, correct? I'll follow this up with a fallback.

Reply:
> hasTranslation would be relevant here, correct? 

Correct. Thanks for following it up :)